packages/gql/src/balancer-gauges/gauge.ts

```diff
@@ -7,3 +7,34 @@ export const gaugeSymbol = gql`
     }
   }
 `;
+
+export const poolsWithGauge = gql`
+  query PreferentialGauge {
+    pools(first: 1000, where: { gauges_: { gauge_not: null } }) {
```

Comment:
```suggestion
    pools(first: 1000, where: { gauges_not: null }) {
```

Reply:
I believe this is already enough - naming here doesn't help much but we need pools which have liquidity gauges (so it can't be null) but it's fine if the gauge attribute is empty - it simply indicates whether you can vote on that gauge, but that's not what we're looking for here.